pkg/api/v2/types.go

```diff
@@ -1878,3 +1911,17 @@ type ClusterServiceAccount struct {
 	apiv1.ObjectMeta `json:",inline"`
 	Namespace        string `json:"namespace,omitempty"`
 }
+
+// ApplicationDefinitionListItem is the object representing an ApplicationDefinitionListItem.
+// swagger:model ApplicationDefinitionListItem
+type ApplicationDefinitionListItem struct {
+	Name string `json:"name"`
+
+	Spec ApplicationDefinitionListItemSpec `json:"spec"`
+}
+
+// ApplicationDefinitionListItemSpec defines the desired state of ApplicationDefinitionListItemSpec.
+type ApplicationDefinitionListItemSpec struct {
```

Comment:
I assume we expect this struct may grow over time, hence only one field for now.